spaces/S000138/properties/P000164.md

```diff
@@ -4,4 +4,4 @@ property: P000164
 value: true
 ---
 
-$X$ has cardinality bounded above by $\aleph_\omega^\omega$, which is below the first inaccessible cardinal and thus $|X|$ is non-measurable.
+$|X| \leq \aleph_\omega$ and $\aleph_\omega$ is smaller than any measurable cardinal
```

Comment:
```suggestion
$|X| \leq \aleph_\omega^\omega$ and $\aleph_\omega^\omega$ is smaller than every measurable cardinal.
```
Raising to the power of $\omega$ here is very much necessary - by [König's theorem](https://en.wikipedia.org/wiki/K%C3%B6nig%27s_theorem_(set_theory)), it has cardinality strictly larger than $\aleph_\omega$. This is actually why it is consistent that $|X| = \mathfrak{c}$ - if $|X| = \aleph_\omega$ it would have been inconsistent to have $|X| = \mathfrak{c}$ as the continuum can be proven to be not $\aleph_\omega$.

Reply:
My bad. GCH even implies $\aleph_\omega^\omega = \aleph_{\omega+1}$